src/lib/components/Spinner.svelte

```diff
@@ -2,7 +2,9 @@
 	import { fade } from 'svelte/transition';
 </script>
 
-<div class="spinner" transition:fade={{ duration: 250 }} />
+<div class="spinner-container">
+	<div class="spinner" transition:fade={{ duration: 250 }} />
+</div>
 
 <style>
 	@keyframes spinner {
```

Comment:
Instead of adding a spinner-container, we could just add the translate style to the `spinner` animation. We can combine both the translate and rotate styles in one transform style. Try doing making a separate `from` keyframe and then adjusting the `to` keyframe.

Reply:
Also, sorry for the late reply!

Reply:
I've went back to your described approach, of making translate -50%, -50%, please review!